open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -480,6 +488,79 @@ def list_hackathon_awards(self, hackathon):
         awards = hackathon.award_contents.order_by(Award.level.desc()).all()
         return [a.dic() for a in awards]
 
+    def get_hackathon_notice(self, notice_id):
+        hackathon_notice = self.db.get_object(HackathonNotice, notice_id)
+        return hackathon_notice.dic()
+
+    def create_hackathon_notice(self, hackathon, body):
```

Comment:
My suggestion:
- ask for `hackathon_id` rather than `hackathon` since I believe some notices may not connect to certain hackathon. For example, template lib
- If it's designed for all modules(will be invoked by other modules), please add explicit parameter for those required such as `event` and `content`. Otherwise I have to understand the body of this method before calling it somewhere else
- Please leave more documents. Specially about the `event` and `type` which may confuse other people